src/app/components/Points/DelegationPoints.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import { useMediaQuery } from "usehooks-ts";
+
+import { useDelegationsPoints } from "@/app/context/api/DelegationsPointsProvider";
+
+interface DelegationPointsProps {
+  stakingTxHash: string;
+}
+
+export const DelegationPoints: React.FC<DelegationPointsProps> = ({
+  stakingTxHash,
+}) => {
+  const { delegationPoints, isLoading } = useDelegationsPoints();
+  const isMobile = useMediaQuery("(max-width:1000px)");
```

Comment:
We might have issues using JavaScript for media queries
This is why all the previous solutions contained pure CSS solutions to check whether or not we are on mobile or the desktop. This is the first introduction of `useMediaQuery`

If possible, please use the tailwind breakpoints